test/e2e/specs/wp-inline-help-support-search-spec.js

```diff
@@ -81,19 +89,19 @@ describe( `[${ host }] Inline Help support search: (${ screenSize }) @parallel`,
 			assert.equal(
 				resultsCount <= 5,
 				true,
-				`Too many search results displayed. Should be less than or equal to 5 (was ${ resultsCount }).`
+				`Too many search results displayed. Should be less than or equal to 5.`
```

Comment:
Minor: I wonder whether the int values for the number of results should be stored on the `SupportSearch` helper component as static properties rather than sprinkling across the tests?

Reply:
I'll leave it off this time, but that sounds like a good thing for me to keep in mind for my next round of e2e tests!